{"uuid": "2e957398-a029-48cb-a23f-e3fe1aa4e77b", "children": ["6567bb62-529b-4d1e-b328-095e27b81f59", "b60cfba7-42bb-422d-b65b-7162ea5faa90", "ef2a8467-0ef6-4432-8c08-d463f3f92fbe", "c0b5cb1f-d63a-45a5-8b89-a41da72a77a3"], "befores": [{"name": "_Class__pytest_setup_class", "status": "broken", "statusDetails": {"message": "yaml.parser.ParserError: while parsing a flow mapping\n  in \"../api/api.yaml\", line 131, column 9\nexpected ',' or '}', but got '{'\n  in \"../api/api.yaml\", line 133, column 24\n", "trace": "  File \"/usr/local/lib/python3.7/site-packages/pluggy/callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"/usr/local/lib/python3.7/site-packages/_pytest/fixtures.py\", line 964, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/usr/local/lib/python3.7/site-packages/_pytest/fixtures.py\", line 788, in call_fixture_func\n    res = next(it)\n  File \"/usr/local/lib/python3.7/site-packages/_pytest/python.py\", line 703, in xunit_setup_class_fixture\n    _call_with_optional_argument(func, self.obj)\n  File \"/usr/local/lib/python3.7/site-packages/_pytest/python.py\", line 634, in _call_with_optional_argument\n    func(arg)\n  File \"/Users/dengying/PycharmProjects/gm/testcase/test_api.py\", line 14, in setup_class\n    cls.login = Testapi().login()\n  File \"/Users/dengying/PycharmProjects/gm/api/api_face.py\", line 16, in __init__\n    self.data = self.api_load(\"../api/api.yaml\")\n  File \"/Users/dengying/PycharmProjects/gm/api/base_api.py\", line 33, in api_load\n    return self.yaml_load(path)\n  File \"/Users/dengying/PycharmProjects/gm/api/base_api.py\", line 29, in yaml_load\n    return yaml.safe_load(f)\n  File \"/usr/local/lib/python3.7/site-packages/yaml/__init__.py\", line 162, in safe_load\n    return load(stream, SafeLoader)\n  File \"/usr/local/lib/python3.7/site-packages/yaml/__init__.py\", line 114, in load\n    return loader.get_single_data()\n  File \"/usr/local/lib/python3.7/site-packages/yaml/constructor.py\", line 49, in get_single_data\n    node = self.get_single_node()\n  File \"/usr/local/lib/python3.7/site-packages/yaml/composer.py\", line 36, in get_single_node\n    document = self.compose_document()\n  File \"/usr/local/lib/python3.7/site-packages/yaml/composer.py\", line 55, in compose_document\n    node = self.compose_node(None, None)\n  File \"/usr/local/lib/python3.7/site-packages/yaml/composer.py\", line 84, in compose_node\n    node = self.compose_mapping_node(anchor)\n  File \"/usr/local/lib/python3.7/site-packages/yaml/composer.py\", line 133, in compose_mapping_node\n    item_value = self.compose_node(node, item_key)\n  File \"/usr/local/lib/python3.7/site-packages/yaml/composer.py\", line 84, in compose_node\n    node = self.compose_mapping_node(anchor)\n  File \"/usr/local/lib/python3.7/site-packages/yaml/composer.py\", line 133, in compose_mapping_node\n    item_value = self.compose_node(node, item_key)\n  File \"/usr/local/lib/python3.7/site-packages/yaml/composer.py\", line 84, in compose_node\n    node = self.compose_mapping_node(anchor)\n  File \"/usr/local/lib/python3.7/site-packages/yaml/composer.py\", line 127, in compose_mapping_node\n    while not self.check_event(MappingEndEvent):\n  File \"/usr/local/lib/python3.7/site-packages/yaml/parser.py\", line 98, in check_event\n    self.current_event = self.state()\n  File \"/usr/local/lib/python3.7/site-packages/yaml/parser.py\", line 550, in parse_flow_mapping_key\n    \"expected ',' or '}', but got %r\" % token.id, token.start_mark)\n"}, "start": 1588931570647, "stop": 1588931570671}], "start": 1588931570647, "stop": 1588931571050}